llarp/service/sendcontext.cpp

```diff
@@ -26,13 +26,15 @@ namespace llarp
     bool
     SendContext::Send(std::shared_ptr<ProtocolFrame> msg, path::Path_ptr path)
     {
+      if (not path->IsReady())
+        return false;
       if (m_SendQueue.empty() or m_SendQueue.full())
```

Comment:
`if empty` makes sense here, as that means we don't currently have a flush queued.  `if full`, however, makes little sense with the below change from `call now` to `call soon`.